protocol/src/main/scala/sbt/protocol/Serialization.scala

```diff
@@ -46,7 +53,44 @@ object Serialization {
   /**
    * @return A command or an invalid input description
    */
-  def deserializeEvent(bytes: Seq[Byte]): Either[String, EventMessage] =
+  def deserializeEvent(bytes: Seq[Byte]): Either[String, Any] =
```

Comment:
Not really. Unless we limit our events to descend from some common trait we have to use Any here.